docs/core/porting/libraries.md

```diff
@@ -0,0 +1,350 @@
+
+---
```

Comment:
I think this needs to come in the first line. it's not rendering the usual table.